### PR TITLE
Some progress on the input page. No functionality just bare bones

### PR DESCRIPTION
**Pull request by alexandrutatarciuc**
Initial design of the input page
Added snack bar
All functionality is mocked

### CODE REVIEW

**Comment by alexandrutatarciuc** [comment]
image

**Comment by alexandrutatarciuc** [comment]
@ronnigaba if you have any suggestions/remarks let me know.
I think it's a good start for now.
I would like to create a TeamCard component because there's some repetitive code, but I am unsure what functionality it should expose other than a parameter `@bind-Score`. I think that we could start adding some functionality and then will see

File: BoyumFoosballStats-2.0/Pages/_Host.cshtml [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" rel="stylesheet"/>
     <link href="_content/MudBlazor/MudBlazor.min.css" rel="stylesheet"/>
     <link href="css/site.css" rel="stylesheet"/>
-    <link href="BoyumFoosballStats_2.styles.css" rel="stylesheet"/>
+    @* <link href="BoyumFoosballStats_2.styles.css" rel="stylesheet"/> *@
```
Comment (ronnigaba) [inline BoyumFoosballStats-2.0/Pages/_Host.cshtml:19]: Should this be removed entirely?

Comment (alexandrutatarciuc) [reply BoyumFoosballStats-2.0/Pages/_Host.cshtml:19]: Oh, I had a weird styling issue and was trying to figure where it comes from. This is responsible for scoped css, I can see that we might use it. Idk...

**Review by ronnigaba — APPROVED** [review]

**Comment by ronnigaba** [comment]
I definitely think it makes sense to make a child component, even if we don't have too much functionality there yet.
If we add more buttons, swapping etc. in the future, then we'll need it anyway!

**Pull request merged**